cmd/lock-instrument_test.go

```diff
@@ -367,6 +368,9 @@ func TestNsLockMapStatusBlockedToRunning(t *testing.T) {
 		},
 	}
 
+	// Initialize the locks.
+	initNSLock(false)
```

Comment:
why is this needed?

Reply:
> why is this needed?

Umm, if I don't put this, TestNsLockMapStatusBlockedToRunning will fail.

This is also done in other similar unit tests. But frankly, I don't know too much low level to fully confirm why this is needed.

Reply:
Don't know we shouldn't need to do this.. there is something else going wrong.